lib/api/v1/controllers/keywords_test.go

```diff
@@ -42,8 +48,11 @@ var _ = Describe("KeywordsController", func() {
 
 		Context("Given INVALID payload", func() {
 			It("returns the unprocessable entity status", func() {
-				fabricators.FabricateUser("test@gmail.com", "123456")
-				ctx, resp := test.MakeMultipartRequestRequest("/keywords/upload", "keywords/invalid.csv", "text/csv")
+				user := fabricators.FabricateUser("test@gmail.com", "123456")
+				headers := http.Header{}
+				headers.Set("Authorization", "Bearer "+helpers.GenerateToken(user.Base.ID.String()))
+				ctx, resp := test.MakeMultipartRequestRequest("/keywords/upload", "keywords/invalid.csv", "text/csv", headers)
+				middlewares.HandleAuthenticatedRequest()(ctx)
```

Comment:
Should we define the needed helpers directly? So we won't duplycate this code in every tests?

For example: in https://github.com/nimblehq/varun-api/blob/develop/lib/api/v1/controllers/jobs_test.go#L48

```
						pilot := FabricatePilot(test.ThaiPhoneNumber())
						farmer := FabricateFarmer(test.ThaiPhoneNumber())
						now := time.Now()
						_ = FabricateJobFor(farmer, pilot, &now, "all-day", models.AcceptedStatus)

						ctx, resp := test.MakeAuthenticatedRequest(http.MethodGet, "/jobs", nil, pilot)
						jobsController := controllers.JobsController{}

						jobsController.Index(ctx)

						Expect(resp.Code).To(Equal(http.StatusOK))
```

Only 1 line is needed in the tests, so it's easiers to write and read :)